doc/release/RELEASE-NOTES.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html). The format is based on the `RELEASE-NOTES-TEMPLATE.md` file.
 
+## Release 1.23.0
+
+## Introduction
+* Product name: Open Supply Hub
+* Release date: November 2, 2024
+
+### Code/API changes
+* introduce throttling for tiles/* endpoins 300/minute 
```

Comment:
It would be better to write it as a statement rather than as an imperative. There is a missing 't' in the word 'endpoins'

Reply:
Pls, use capitalized 'i' in the 'introduce' word.

Reply:
Done please check